frontend/src/app/app.module.ts

```diff
@@ -113,9 +113,13 @@ const appRoutes: Routes = [
     path: 'password-reset',
     component: PasswordResetComponent,
   },
+  {
+    path: 'page-404',
+    component: Page404Component,
+  },
   {
     path: '**',
-    component: RedirectComponent
+    component: Page404Component,
```

Comment:
I'm not fully up to date, but won't it break those hackish redirects to django views that are not yet in angular? Or maybe all the views have already been migrated from django to angular?

So if everything is now in angular, then cool.

However, if not  (and we still need this `RedirectComponent` under `**` pattern), then IMO it's better to wait until all the views are transfered. Currently 404 in angular would have to look like:
1. Angular does not recognize route, so redirects to django as it might be django view
2. Django does not recognize route, so redirects back to angular 404 page
3. Angular displays 404

Well, that's ugly enough to leave 404 in django for now IMO.

Reply:
I thought It would as we have a few views still in django (for eg. contact with administrator), checked and it looked like everything was working fine, but now I've checked again and it actually breaks redirect in one case I forgot to check before - my account view (me), but I believe It can be easliy fixed by adding `this.djangoRoot = environment.djangoRoot;` in `header.component.ts` (checked, works).